diff --git a/src/Plugin/Field/FieldWidget/TermReferenceTree.php b/src/Plugin/Field/FieldWidget/TermReferenceTree.php
index c02181c..ca68117 100644
--- a/src/Plugin/Field/FieldWidget/TermReferenceTree.php
+++ b/src/Plugin/Field/FieldWidget/TermReferenceTree.php
@@ -66,7 +66,7 @@ public static function validateTermReferenceTreeElement(&$element, FormStateInte
 
           // If the element is leaves only and select parents is on, then automatically
           // add all the parents of each selected value.
-          if ($element['#select_parents'] && $element['#leaves_only']) {
+          if (isset($element['#select_parents']) && $element['#leaves_only']) {
             foreach ($child['#parent_values'] as $parent_tid) {
               if (!in_array(array($element['#value_key'] => $parent_tid), $value)) {
                 array_push($value, array($element['#value_key'] => $parent_tid));
